presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergPageSink.java

```diff
@@ -328,8 +327,7 @@ private Metrics getMetrics(WriteContext writeContext)
             case PARQUET:
                 return ParquetUtil.fileMetrics(HadoopInputFile.fromPath(writeContext.getPath(), jobConf), MetricsConfig.getDefault());
             case ORC:
-                return writeContext.getWriter().getMetrics()
-                        .orElseThrow(() -> new VerifyException("Iceberg ORC file writers should return Iceberg metrics"));
+                return writeContext.getWriter().getMetrics();
```

Comment:
This entire `getMetrics()` method is no longer needed. (and should be removed since we're reading the metrics twice for Parquet)

Reply:
Removed.